src/test/java/com/puppycrawl/tools/checkstyle/ant/CheckstyleAntTaskTest.java

```diff
@@ -66,6 +69,9 @@ public class CheckstyleAntTaskTest extends AbstractPathTestSupport {
     private static final String NOT_EXISTING_FILE = "target/not_existing.xml";
     private static final String FAILURE_PROPERTY_VALUE = "myValue";
 
+    @TempDir
+    public File temporaryFolder;
```

Comment:
What do we need this for?

Reply:
My bad, we don't require it, earlier instead of using `CheckstyleAntTaskLogStub` I was logging to a temporary file, removed it now, done.